data-raw/data-generator.R

```diff
@@ -49,6 +49,19 @@ ggsci_db$"jama"$"default" <- c(
   "Makara" = "#80796B"
 )
 
+# Color palette inspired by plots in BMJ (http://technology.bmj.com/living-style-guide/colour.html)
+ggsci_db$"bmj"$"default" <- c(
+  "Blue" = "#2A6EBB", 
+  "Grey" = "#747678",
+  "Yellow" = "#F0AB00", 
+  "Orange" = "#E37222",
+  "Red" = "#CD202C", 
+  "Pink" = "#C50084",
+  "Purple" = "#7D5CC6", 
+  "Aqua" = "#00B2A9",
+  "Green" = "#69BE28"
```

Comment:
Please rearrange the color order by

blue, yellow, pink, purple, orange, green, aqua, red, grey.

See Figure 2 in this paper for example: https://doi.org/10.1136/bmj-2023-077797